modules/apigee-x-bridge-mig/main.tf

```diff
@@ -77,7 +77,7 @@ resource "random_id" "fw" {
 
 resource "google_compute_firewall" "allow_glb_to_mig_bridge" {
```

Comment:
I'd rather move the firewall out of the mig module and make it agnostic of shared vpc setups. WDYT?